csaf-validation/src/main/kotlin/io/github/csaf/sbom/validation/Requirement.kt

```diff
@@ -37,18 +47,22 @@ fun allOf(vararg requirements: Requirement): Requirement {
     return AllOf(requirements.toList())
 }
 
-internal class AllOf(var list: List<Requirement>) : Requirement {
+internal class AllOf(private val list: List<Requirement>) : Requirement {
```

Comment:
Documentation

Reply:
I intentionally did not document the internal classes (I set them to private now). The public functions to create them are documented.